### DISCUSSION TITLE
403 forbidden error in serving the pages in sandwich menu

### DISCUSSION THREAD

**Discussion by Akanksha0919** [mkdocs/mkdocs#3294] [Q&A]
I am facing 403 forbidden issue with mkdocs
the individual htmls for different modules are working fine but the sandwich icon showing all "pages" gives 403 forbidden error
image

![image](https://github.com/mkdocs/mkdocs/assets/65400521/79f7f40c-9156-4c2b-ad2e-07071335d07b)

all the md pages have their own index.html which works fine but the page links on sandwich icon don't work. what do you think , is this the issue of permissions? because in mkdocs.yml file i have the following code :
site_name: dynamic pricing documentation

theme:
name: "material"

nav:

Intro Docs: index.md
evaluation documentation: evaluation.md
metrics documentation: metrics.md
plugins:

mkdocstrings

I think the only thing that is crucial to connecting to these md pages is the nav and plugin: mkdocstrings

how can i resolve this 403 forbidden error...
is this issue of permissions from server side or can I resolve this using some site_url or repo_url or something else?
i dont want anything complicated, just want a static html which have links to all pages work properly.

**Comment by pawamoy** [comment]
As told in the other issue, to me it looks like an issue with how you serve your docs, i.e. on GCP hosted Jupyterlab.

What do you mean by "the pages work fine"? How do you access them? What URLs do you type? Are they the same as the ones in the menu? What is the actual setup: how do you deploy your docs on Jupyterlab?

**Comment by Akanksha0919** [comment]
I resolved the issue.. I was reading the documentation of mkdocs and their is addition I made to mkdocs.yml file. I added 
use_directory_urls: false, this directly links to the md files and not the directories. 

The difference of using this use_directory_urls: false can be seen after mkdocs build. 

1. after using use_directory_urls: false , my site directory folder looks like this: 
- site
    - evaluation.html
    - metrics.html
    - index.html
    - 404.html
    - ... so on...........  

2. without using use_directory_urls: false, my site directory structure looks like this:
- site
    - evaluation (folder)
           - evaluation.html   
   - metrics
          - metrics.html
  - index.html
  - 404.html
  - so on......

for me 1st one works and I dont get 403 anymore. 

i will close the issue now. 
@pawamoy thank you for your answers and support😊

**Comment by pawamoy** [comment]
Glad you found the solution, thanks for sharing it and closing the issue 🙂
